lib/turbo_tests/runner.rb

```diff
@@ -254,12 +259,17 @@ def handle_messages
             break
           end
         when "message"
-          @reporter.message(message[:message])
+          if message[:message].include?("An error occurred") || message[:message].include?("occurred outside of examples")
```

Comment:
We have to check message contents because RSpec outputs errors outside of examples as regular messages which we format in `TurboTests::JsonRowsFormatter#message`.

Ref `RSpec::Core::Reporter#notify_non_example_exception`: https://github.com/rspec/rspec-core/blob/5699fcdc4723087ff6139af55bd155ad9ad61a7b/lib/rspec/core/reporter.rb#L163-L170

Ref `TurboTests::JsonRowsFormatter#message`: https://github.com/serpapi/turbo_tests/blob/bb814e1a65ea5cf467e83a9d7aacb093802a5c82/lib/turbo_tests/json_rows_formatter.rb#L97-L102

Errors outside of examples begin with `An error occurred` or `While loading`:
* https://github.com/rspec/rspec-core/blob/5699fcdc4723087ff6139af55bd155ad9ad61a7b/lib/rspec/core/hooks.rb#L383
* https://github.com/rspec/rspec-core/blob/5699fcdc4723087ff6139af55bd155ad9ad61a7b/lib/rspec/core/example.rb#L660
* https://github.com/rspec/rspec-core/blob/5699fcdc4723087ff6139af55bd155ad9ad61a7b/lib/rspec/core/configuration.rb#L2142
* https://github.com/rspec/rspec-core/blob/5699fcdc4723087ff6139af55bd155ad9ad61a7b/lib/rspec/core/configuration.rb#L2159

And end with `occurred outside of examples`:
* https://github.com/rspec/rspec-core/blob/5699fcdc4723087ff6139af55bd155ad9ad61a7b/lib/rspec/core/notifications.rb#L332